jerry-core/vm/vm.c

```diff
@@ -1660,6 +1660,18 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           }
           continue;
         }
+        case VM_OC_PROP_GET:
+        {
+          result = vm_op_get_value (left_value, right_value);
+
+          if (ECMA_IS_VALUE_ERROR (result))
+          {
+            goto error;
+          }
+
+          *stack_top_p++ = result;
+          goto free_both_values;
+        }
```

Comment:
Isn't the `if (opcode >= CBC_PUSH_PROP_REFERENCE && opcode < CBC_PRE_INCR)` checks below are always true after this change?

Reply:
Good catch!